api/include/opentelemetry/trace/span.h

```diff
@@ -43,8 +43,13 @@ struct StartSpanOptions
   core::SystemTimestamp start_system_time;
   core::SteadyTimestamp start_steady_time;
 
+  // Explicitely set the parent of a Span.
+  //
+  // This defaults to an invalid span context. In this case, the Span is
+  // automatically parented to the currently active span.
+  SpanContext parent = SpanContext::GetInvalid();
```

Comment:
Defaults to invalid value may look a little weird, would `GetEmpty()` a better name?

Reply:
I was thinking about that, but then decided against introducing an _empty_ span context. In the current terminology, a span context can either be _valid_ or _invalid_ (it has an `IsValid` method). So `SpanContext::GetInvalid()::IsValid() == true` I think is more idiomatic than `SpanContext::GetEmpty()::IsValid() == true`. And here, an invalid span context is used like a null pointer (indicating that no parent span context is set).

But I don't have a very strong opinion on that, I'd be willing to go with `GetEmpty()` if folks feel that's clearer.

Reply:
I too don't have strong opinion on this, although having two different methods `GetEmpty()` and `GetInvalid()` returning invalid span context is more confusing to me.